### DISCUSSION TITLE
All Account Pages (/dashboard, /account, /account/teams) 404 after creating account

### DISCUSSION THREAD

**Discussion by krauzer** [vercel/community#5355] [Account Issue]
### Question

I created an account linked to Github under Pro but then deleted the team. After that I can't navigate at all through the account pages. They produce 404s and an error in the console. I can't even delete the account to try to recreate it because the admin page 404s. Same issue as this: #5341 

Sequence of Events: 
1. Invited to join a team via an Email
2. Upon clicking the link, I was prompted to sign up. 
3. I selected Pro, thinking it would link me to the team. 
4. I signed up using Github. 
5. It created a new team instead of linking me to the invited team and put me under the 14 day trial for Pro. 
6. I deleted the new team, which seemed to have downgraded me to the Hobby Account. 
7. I started getting 404 pages for all the account pages, so I can't do anything including trying to actually join the team I was invited to. Nor can I delete my account to see if recreating it fixes it. 

The issue might stem from a 500 error that shows up in the Console for this route:  `https://vercel.com/api/toolbar/comments/inbox/count?status=unresolved`

<img width="1486" alt="vercel_async_error" src="https://github.com/vercel/community/assets/6069550/31609f02-9f27-490e-9e33-1217d2a6b72b">

### Image of 404 

![vercel](https://github.com/vercel/community/assets/6069550/38ef53db-88fc-4785-aa52-e28b76614965)

I suspect having deleted the only team might have had something to do with it?

**Comment by amyegan** [comment]
Hi Dylan. Thanks for reporting this. I'm looking into it now

**Comment by krauzer** [comment]
@amyegan I fixed it by creating a new dummy pro trial team. 

I was then able to create a hobby team, delete the dummy team, and accept the invite. 

It seems like a bug in the system where you're able to delete your only team when using pro trial and not having a team causes all the account pages to crash. It's easy to recreate. 

Please alert your dev team to this bug.

**Reply by amyegan** [reply]
I'm glad you already found a solution! Thanks for sharing your steps. I'll let the team know so we can get this bug fixed.
